### PR TITLE
[BUG][ConvertLayout] Fix qnn.conv2d layout conversion too many values to unpack

### PR DESCRIPTION
**Pull request by lhutton1**
This patch follows a previous bugfix in #6419. I made a very simple oversight for qnn.conv2d in that tinfos also contains qnn parameters. Therefore, we need to extract data_info and weight_info differently.

cc @leandron @anijain2305 @JoeyTPChou

### CODE REVIEW

**Review by comaniac — APPROVED** [review]
LGTM

**Review by anijain2305 — APPROVED** [review]

**Comment by masahi** [comment]
please wait until we merge https://github.com/apache/incubator-tvm/pull/6448 first.

**Comment by masahi** [comment]
Thanks @lhutton1 @anijain2305 @comaniac @leandron

**Pull request merged**
